src/lib/ledgers/five-bells-ledger.js

```diff
@@ -146,18 +134,24 @@ FiveBellsLedger.prototype._autofund = function * () {
       connector: config.getIn(['server', 'base_uri']),
       password: this.credentials.password,
       fingerprint: this.credentials.fingerprint
-    },
-    cert: admin.cert,
-    ca: admin.ca,
-    key: admin.key
-  }, 'could not create account at ledger ' + this.id)
+    }
+  }, 'could not create account at ledger ' + this.id, admin)
 }
 
-function * request_retry (opts, error_msg) {
+function * request_retry (opts, error_msg, credentials) {
   let delay = backoffMin
   while (true) {
     try {
-      let res = yield request(opts)
+      let res = yield request(lodash.defaults(opts, {
+        auth: credentials && {
```

Comment:
Is it `credentials.username/password` or `credentials.user/pass`? Pick one. Also, instead of `credentials && {...` it should be `credentials.password && {...` so that we don't send basic auth of undefined:undefined when using client cert auth.